### DISCUSSION TITLE
[BO] Carrier - "Transit time" - should be optional

### DISCUSSION THREAD

**Discussion by ShaiMagal** [PrestaShop/PrestaShop#37460] [Ideas]
### Prerequisites

- [x] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [x] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Now is "Transit time" required, but why?
There is a lot of real case, where you don't want to show any text, just only carrier name..

It's possibility how to "workaround" this, you should use empty spaces to "Transit time" or something like that, but why to do to this "hack"?

**Make this optional, not required. :)** 

![Image](https://github.com/user-attachments/assets/fa17c495-64c0-40da-b43a-becfaf1d27b7)

### Expected behavior

Transit time should be optional only.

### Steps to reproduce

1. go to carriers section in BO
2. edit any carrier
3. try to continue with empty "Transit time".
4. you got error message.

### PrestaShop version(s) where the bug happened

9.0.0

### PHP version(s) where the bug happened

8.3

### Your company or customer's name goes here (if applicable).

https://www.openservis.cz/

**Comment by kpodemski** [comment]
It’s not a new thing.

What are the scenarios you know about that you could share?

**Comment by Hlavtox** [comment]
@kpodemski I have for example never ever ever needed the transit time. "Personal pickup at our shop in my city" - what transit time should I write here?

**Comment by kpodemski** [comment]
@Hlavtox 

"Your order will be available to pickup in X hours"

**Reply by BenTen** [reply]
Yes we can find a lof way to write but the problem is we don't have choice, we have to write something.

I never understood why this option is mandatory and not "Speed grade".
It's nice to have but not mandatory.

But yes, it's not from PS 9, at least in 1.7 is already like that.
